dao/beacon/attestations.gen.go

```diff
@@ -0,0 +1,34 @@
+// Code generated by gorm.io/gen. DO NOT EDIT.
+// Code generated by gorm.io/gen. DO NOT EDIT.
+// Code generated by gorm.io/gen. DO NOT EDIT.
+
+package beacon
+
+import (
+	"time"
+)
+
+const TableNameAttestation = "attestations"
+
+// Attestation mapped from table <attestations>
+type Attestation struct {
+	SlotNumber      int64     `gorm:"column:slot_number;primaryKey" json:"slot_number"`
+	Index           int32     `gorm:"column:index;primaryKey" json:"index"`
+	BeaconBlockRoot string    `gorm:"column:beacon_block_root" json:"beacon_block_root"`
+	AggregationBits string    `gorm:"column:aggregation_bits" json:"aggregation_bits"`
+	AttestationSlot int64     `gorm:"column:attestation_slot" json:"attestation_slot"`
```

Comment:
please double check the schema and data types...
i do not review them...

Reply:
sure